.dotnet/src/Custom/Chat/ChatCompletion.cs

```diff
@@ -61,6 +61,9 @@ public partial class ChatCompletion
     // CUSTOM: Flattened choice message property.
     public ChatFunctionCall FunctionCall => Choices[0].Message.FunctionCall;
 
+    [CodeGenMember("ServiceTier")]
+    public ServiceLatencyTierOutcome? LatencyTierOutcome { get; }
```

Comment:
We should keep the original name "ServiceTier" for this property. I don't think renaming it adds enough value. It should also have the same name here as the name used in StreamingChatCompletionUpdate.